app/models/project.rb

```diff
@@ -50,6 +51,11 @@ def budget_status
     budget - hours.sum(:value) if budget
   end
 
+  def update_user_activity
+    user_activity = DateTime.now
+    save
+  end
```

Comment:
Ok, no problem! Discard this pull request, it was only an idea :) All wil be ok with touch only.